validation/src/main/scala/hmda/validation/engine/lar/syntactical/LarSyntacticalEngine.scala

```diff
@@ -6,19 +6,19 @@ import hmda.validation.context.ValidationContext
 import hmda.validation.engine.Syntactical
 import hmda.validation.engine.lar.LarCommonEngine
 import hmda.validation.rules.EditCheck
-import hmda.validation.rules.ts.syntactical.S025
+import hmda.validation.rules.ts.syntactical.{ S020, S025 }
 import hmda.validation.rules.lar.syntactical._
 
 trait LarSyntacticalEngine extends LarCommonEngine with ValidationApi {
 
   def checkSyntactical(lar: LoanApplicationRegister, ctx: ValidationContext): LarValidation = {
-    val checksToRun: List[EditCheck[LoanApplicationRegister]] = List(
+    val checksToRun: List[EditCheck[_ >: LoanApplicationRegister]] = List(
       S010,
       S020,
       S025.inContext(ctx),
       S205
     )
-    val checks = checksToRun.map(check(_, lar, lar.loan.id, Syntactical))
+    val checks = checksToRun.map(check[LoanApplicationRegister](_, lar, lar.loan.id, Syntactical))
```

Comment:
Interesting, and a bit weird. The other LAR engines didn't require it? I guess that's because they run "normal" edit checks? (guessing, haven't checked)